README.md

```diff
@@ -108,7 +108,7 @@ You should see the following:
 
 You can then try different queries against a pre-defined set of indexed articles related to Game of Thrones.
 
-**Note**: The following containers are started as part of this demo:
+**Note**: This demo starts the following containers:
```

Comment:
This one is probably about rewriting the "passive voice" by Grammarly? I'd leave it as-is here, though.

Reply:
Yes, it is. Reverted.